shared/elements/gaia-site-icon/script.js

```diff
@@ -334,7 +346,8 @@ window.GaiaAppIcon = (function(exports) {
       if (!this._hasUserSetIcon) {
         // Process the image
         var canvas = document.createElement('canvas');
-        canvas.width = canvas.height = this._size;
+        var size = this.size * window.devicePixelRatio;
```

Comment:
Seems like we fetch this adjusted size multiple times. Might be worth having a getter or function for it.